### PR TITLE
Use regular pip install for torch+cpu package; Deprecate light-the-torch

### PR DESCRIPTION
**Pull request by shifucun**
light-the-torch can not specify the torch version, which is a tricky when matching sentence-transformer and other ml library packages

### CODE REVIEW

**Review by chejennifer — APPROVED** [review]
cool! Curious what the difference between light-the-torch and regular pip install is?

**Comment by shifucun** [comment]
> cool! Curious what the difference between light-the-torch and regular pip install is?

light-the-torch can not specify the torch version, which is a tricky when matching sentence-transformer and other ml library packages.

File: build/web_compose/Dockerfile [hunk 1]
```diff
@@ -64,8 +64,8 @@ RUN pip3 install -r /workspace/server/requirements.txt
 COPY nl_app.py /workspace/nl_app.py
 COPY deploy/nl/. /datacommons/nl/
 COPY nl_server/requirements.txt /workspace/nl_server/requirements.txt
-RUN pip install --upgrade pip setuptools light-the-torch
-RUN ltt install torch --cpuonly
+RUN pip install --upgrade pip
+RUN pip3 install torch==2.2.1 --extra-index-url https://download.pytorch.org/whl/cpu
```
Comment (pradh) [inline build/web_compose/Dockerfile:68]: To check, this will increase the docker image (that was the original motivation for light-the-torch) how much?

**Review by pradh — APPROVED** [review]
Thanks!

File: build/web_compose/Dockerfile (hunk 1 above)
Comment (shifucun) [reply build/web_compose/Dockerfile:68]: Nope, this also installs cpu only torch and the size is same.

Comment (shifucun) [reply build/web_compose/Dockerfile:68]: I should have found this way instead of light-the-torch :)

**Pull request merged**